breacharbiter.go

```diff
@@ -849,10 +849,6 @@ func (bo *breachedOutput) BuildWitness(signer lnwallet.Signer, txn *wire.MsgTx,
 	return bo.witnessFunc(txn, hashCache, txinIdx)
 }
 
-// Add compile-time constraint ensuring breachedOutput implements
```

Comment:
I removed the interface because it was not used anywhere. But indeed, I can also see the `breacharbiter` use an (its own) instance of the sweeper. Converted to `Input` types.